test/browser/packages/cfpb-expandables.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable no-undef */
+
+describe( 'basic cfpb expandable', () => {
```

Comment:
Trivial: top-level `describe` should capitalize the first word `Basic`

Reply:
And `CFPB` 😝